src/otx/cli/__init__.py

```diff
@@ -11,7 +13,10 @@ def main() -> None:
 
     This function is a single entry point for all OTX CLI related operations:
     """
+    start = time()
```

Comment:
Is that  change intended?

Reply:
No but is there any problem with it? @harimkang

Reply:
I don't see a problem with that.